index.bs

```diff
@@ -290,9 +308,9 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 : <dfn>RP ID</dfn>
 :: An identifier for the [RP] on whose behalf a given registration or authentication ceremony is being performed. Scoped
     credentials can only be used for authentication by the same entity (as identified by RP ID) that created and registered
-    them. By default, the RP ID for a WebAuthn operation is set to the [=origin=] specified by the {{WebAuthentication}}
-    object's [=relevant settings object=]. This default can be overridden by the caller subject to certain restrictions, as
-    specified in [[#makeCredential]] and [[#getAssertion]].
+    them. By default, the RP ID for a WebAuthn operation is set to the [=environment settings object/origin=] specified by the
+    [=current settings object=]. This default can be overridden by the caller subject to certain restrictions, as
```

Comment:
wrt `[=current settings object=]` --   given the language a ways below of:
> Let |callerOrigin| be the [=environment settings object/origin=] specified by this {{ScopedCredential}} object's [=relevant settings object=]. 


..is using `[=current settings object=]` here correct?